src/legacy/ui/ui_settings/create_objects_client_stub.ts

```diff
@@ -18,35 +18,23 @@
  */
 
 import sinon from 'sinon';
-import expect from '@kbn/expect';
-import { SavedObjectsClient } from '../../../../../core/server';
+import { SavedObjectsClient } from '../../../../src/core/server';
 
 export const savedObjectsClientErrors = SavedObjectsClient.errors;
 
-export function createObjectsClientStub(type, id, esDocSource = {}) {
+export interface SavedObjectsClientStub {
+  update: sinon.SinonStub<any[], any>;
```

Comment:
Should we switch to Jest mocks while we're at it?

Reply:
I thought to update tests according to modern standards, but realize it will take some time and distract us from:
- migration to TS 
- introducing uiSettings in the NP

I'd suggest postponing this work until we have more resources.